### PR TITLE
Import Linux Kernel typos: letter H

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

**Comment by Gelma** [comment]
Funny, it passed all tests on my laptop.

Can someone else - please - try and check?

**Comment by luzpaz** [comment]
how do you run the test locally ?

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -5592,10 +5594,12 @@ hanshakes->handshakes
 hapen->happen
 hapened->happened
 hapening->happening
+happaned->happened
 happend->happened
 happended->happened
 happenned->happened
 happing->happening
+happne->happens
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: also (and more likely) `happen`

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Fixed

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -5613,6 +5621,7 @@ headquarer->headquarter
 headquater->headquarter
 headquatered->headquartered
 headquaters->headquarters
+headrom->headroom
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Also `bedroom`

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Added.

**Comment by Gelma** [comment]
I'm going to close this PR, and open a new one.

**Pull request closed without merging**